src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
 // write code here
+
+const table = document.querySelector('table');
+const tableBody = document.querySelector('tbody');
+const tablerows = document.querySelectorAll('tbody tr');
+const tablerowsArray = Array.from(tablerows);
+
+table.addEventListener('click', function(evt) {
+  evt.preventDefault();
```

Comment:
The evt.preventDefault() function is unnecessary here. Click events on table headers (th elements) do not have a default action and do not need to be prevented.